facebook_connector.py

```diff
@@ -0,0 +1,40 @@
+from fbchat_archive_parser.parser import MessageHtmlParser
+import fnmatch
+import os
+import pandas as pd
+
+
+def initialize(dump_location='.'):
+    global messages_file
+    fb_message_filename = 'messages.htm'
+    messages_file = os.path.join(dump_location, fb_message_filename)
+    if not os.path.isfile(messages_file):
+        print("Please place the message.htm file at {}".format(
+            os.path.abspath(messages_file)))
+        return
+
+
+def get_cleaned_fully_merged_messages():
+    chats = MessageHtmlParser(path=messages_file).parse()
+    me = chats.user
+    addresses = set()
+    messages = []
+    for thread in chats.threads.itervalues():
+        for participant in thread.participants:
+            if participant is not None and not participant.isspace():
+                addresses.add(participant)
+        for message in thread.messages:
+            if (message.content is not None
```

Comment:
`message.content is not None and message.content` is equivalent to `message.content` since None would always fail `message.content` :)